docs/migration-guides/sfa-ios-v8-to-v9.mdx

```diff
@@ -0,0 +1,128 @@
+let singleFactorAuthArgs = SFAParams(
+    web3AuthClientId: "YOUR_WEB3AUTH_CLIENT_ID",
+    network: .sapphire(.SAPPHIRE_MAINNET)
+)
+// remove-end
+
+// add-start
+let web3AuthOptions = Web3AuthOptions(
+    clientId: "YOUR_WEB3AUTH_CLIENT_ID",
+    web3AuthNetwork: .SAPPHIRE_MAINNET
+)
+// add-end
+
+let singleFactoreAuth = SingleFactorAuth(
+    // remove-next-line
+    params: SFAParams
+    // add-next-line
+    params: web3AuthOptions
+)
+```
+
+### SFAKey is replaced with SessionData
+
+In v9, we removed the `SFAKey` and added the `SessionData` object to return the relveant session
+information like private key, address, user information, and signatures.
+
+```swift
+// remove-next-line
+let sfaKey: SFAKey = try await singleFactoreAuth.connect(loginParams: loginParams)
+// add-next-line
+let sessionData: SessionData = try await singleFactoreAuth.connect(loginParams: loginParams)
+```
+
+### initialize method changes
+
+Starting v9, the `initialize` method will not return any value. To check whether the user already
+has an existing session, please use the `getSessionData` method. The `getSessionData` method will
+return `nil` if the user does not have an existing session.
+
+```swift
+// remove-next-line
+let sfaKey: SFAKey = try await singleFactoreAuth.initialize()
+// add-next-line
+try await singleFactoreAuth.initialize()
+
+// add-start
+let sessionData = singleFactoreAuth.getSessionData()
+if sessionData == nil {
+    // User does not have an existing session
+}
+// add-end
+```
+
+## Additional Features
+
+Apart from the breaking changes, we have added couple of new functions in v9 to improve the
+developer experience.
+
+### logout Method
+
+The `logout` method is added to the SDK to log out the user and clear the session data. Please note,
+that this method only clears the Web3Auth session from local storage.
+
+```swift
+try await singleFactoreAuth.logout()
+```
+
+### getSessionData Method
+
+The `getSessionData` method is added to the SDK to get the session data. You can use this method to
+check whether the user already has an existing session.
+
+#### Usage
+
+```swift
+let sessionData = singleFactoreAuth.getSessionData()
+
+if sessionData == nil {
+    // User does not have an existing session
+}
+```
+
+#### SessionData Response
+
+The `SessionData` has the following four functions to retrive the relevant session information.
+
+| Function Name      | Description                                                               |
+| ------------------ | ------------------------------------------------------------------------- |
+| `getPrivateKey`    | Retrieves the user's private key.                                         |
+| `getPublicAddress` | Retrieves the user's public address.                                      |
+| `getUserInfo`      | Retrieves the user's information like email, name, verifier id, and more. |
+| `getSignatures`    | Retrieves the signatures.                                                 |
```

Comment:
This should be added as inline documentation to the class here:
https://github.com/Web3Auth/single-factor-auth-swift/blob/main/Sources/SingleFactorAuth/SessionData.swift#L5

And then referred by building the documentation in swift.

See here for a reference on how to do this:
https://nshipster.com/swift-documentation/

Reply:
We'll need to think how we want to do this, it's not a straight forward change. We'll explore this in the sprint cc:// @shahbaz17 @yashovardhan